docs/update-locking.md

```diff
@@ -35,7 +59,16 @@ Using the above-mentioned library, the lock can be acquired like in this CoffeeS
 			throw err if err?
 ```
 
-There are other libraries you can use in different languages, for example [this Python library](http://pythonhosted.org/lockfile/lockfile.html#examples).
+#### Python
+
+In Python you can use the for example the [`lockfile` library](http://pythonhosted.org/lockfile/lockfile.html#examples)
```

Comment:
NIT
`In Python you can use the for example the` --> `In Python you can use the`

Reply:
Done